lib/services/gitlab/gitlab_repo_resource.rb

```diff
@@ -2,7 +2,7 @@ class GitlabRepoResource < GitlabResource
   def all
     return @repos if @repos
     prepare_request
-    gitlab_http_get_paginated("#{@service.server_url}/projects") do |repos|
+    gitlab_http_get_paginated("#{@service.server_url}/projects?owned=true") do |repos|
```

Comment:
This will be ignored in `v3` but without it in `v4` we get _all_ public repos

Reply:
Might want to put this as a comment in the code. Maybe with a link to the api changes url (https://docs.gitlab.com/ce/api/v3_to_v4.html)